docs/source/architectureref/certificates_path_list_quorum.md

```diff
@@ -0,0 +1,49 @@
+
+| Path                                                                              | Crypto-material               | Type         |
+|-----------------------------------------------------------------------------------|-------------------------------|--------------|
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | nodekey                       | Public Key   |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | keystore                      | Private Key  |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | db_user                       | Username     |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | dn_password                   | Password     |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | gethpassword                   | Password     |
+
+
+### For RAFT
+
+| Path                                                                             | Crypto-material               | Type         |
+|----------------------------------------------------------------------------------|-------------------------------|--------------|
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                        | nodekey                       | Public Key   |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                        | keystore                      | Private Key  |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                        | db_user                       | Username     |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                        | dn_password                   | Password     |
+| secret/{{`component_ns`}}/crypto/{{ `peer_name` }}/quorum                         | gethpassword                   | Password     |
+
+
+### For Tessera
+
+| Path                                                                           | Crypto-material               | Type         |
+|--------------------------------------------------------------------------------|-------------------------------|--------------|
+| sys/policy/vault-crypto-{{ `component_name` }}-ro                              | tm.pub                        | Public Key   |
+| secret/{{component_ns}}/crypto/{{ `peer_name` }}/transaction                   | tm.key                        | Private Key  |
+
+
+## For Root Certificates
+
+| Path                                                                         | Crypto-material               | Type         |
+|------------------------------------------------------------------------------|-------------------------------|--------------|
+| secret/{{ `component_ns` }}/crypto/{{ `node_name` }}/certs                       | rootca                        | Certificate  |
+| secret/{{ `component_ns` }}/crypto/{{ `node_name` }}/certs                       | ambassadorcrt                 | Certificate  |
```

Comment:
Add description "Certificate chain for Ambassador proxy" 
Also add descriotion for the rootca